easypy/collections.py

```diff
@@ -24,8 +25,38 @@
     from collections import OrderedDict as PythonOrderedDict
 
 
-class ObjectNotFound(LookupError):
-    pass
+def _format_filter_string(preds, filters):
+    returned = []
+    if preds:
+        returned.append("predicates=[%s]" % ", ".join(map(str, preds)))
```

Comment:
This separation seems a bit superficial. 


def x_is_even(obj):
    return obj.x % 2 == 0

print(_format_filter_string([x_is_even], dict(y=5)))


This will print `predicates=[<function x_is_even at 0x7f99923466a8>] filters=[y=5]`. But wouldn't it be better if it'd print: `[x_is_even, y=5]`?